com.ibm.streamsx.topology/opt/python/packages/streamsx/scripts/streamtool.py

```diff
@@ -18,6 +18,9 @@
 import streamsx.topology.context
 from streamsx.rest import Instance
 
+# Return code
+_rc = 0
```

Comment:
Global variables are not generally used, is there a reason this needs to be global?

Reply:
Resolved: removed global variable